iroh-net/src/magicsock/node_map/node_state.rs

```diff
@@ -259,11 +258,7 @@ impl NodeState {
     /// Returns the address(es) that should be used for sending the next packet.
     ///
     /// Any or all of the UDP and relay addrs may be non-zero.
```

Comment:
unrelated I guess, but does this even mean? an address being "non zero"? more go baggage?

Reply:
I read this as "may be `Some`".  Probably the wording has survived from go.

Reply:
i re-worded this a bit.